### PR TITLE
Fix multi-stream team scratch space definition for HIP

### PR DESCRIPTION
**Pull request by masterleinad**
Analogon to #3269 for HIP.

### CODE REVIEW

File: core/unit_test/CMakeLists.txt [hunk 1]
```diff
@@ -139,7 +139,7 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;HIP)
       ${dir}/Test${Tag}_ViewMapping_b.cpp
       ${dir}/Test${Tag}_ViewMapping_subview.cpp
       ${dir}/Test${Tag}_ViewOfClass.cpp
-      ${dir}/Test${Tag}_WorkGraph.cpp
+#      ${dir}/Test${Tag}_WorkGraph.cpp
```
Comment (dalg24) [inline core/unit_test/CMakeLists.txt:142]: Was that intentional?

Comment (masterleinad) [reply core/unit_test/CMakeLists.txt]: Nope, fixed.

**Review by dalg24 — APPROVED** [review]
I wish we were not duplicating the code in the unit test.  (Not asking you to change anything)

**Comment by masterleinad** [comment]
Should this also be part of the patch release?

**Pull request merged**